app/ui/mainwindow2.ui

```diff
@@ -163,6 +163,7 @@
     <addaction name="actionAdd_Frame"/>
     <addaction name="actionDuplicate_Frame"/>
     <addaction name="actionRemove_Frame"/>
+    <addaction name="actionCopy_multiple_Keyframes"/>
```

Comment:
When I started on this, it was only Copy that was in focus. That is why it is called 'copymultiplekeyframesdialog' and 'action_Copy_multiple_Keyframes'.
The last week I have used 'manipulateMultipleFrames' and 'manipulateRange'.
I think I should pick one of these to make it consistent, and maybe manipulateMultipleFrames is the best for consistency with other actions. What do you think?

Reply:
Personally I like manipulateRange. But let’s hear what the others think about this as well.

Reply:
I haven't changed it yet, but after sleeping on it, I'd go for manipulateFrames. Descriptive and short.

Reply:
Anyone else who wants to weigh in?

Reply:
If it's for the dialog as a whole, I think <manipulateFrames> works just fine. If it is just for the copy action only I'd go for <actionCopy_Frames> to maintain consistency with the other functions. 
----
If we wanted to get really specific I'd say we always are manipulating both drawings and frames, at the same time, that is, we are creating the artwork and automatically exposing it to the timeline in order to arrange its timing to create the animation, but since all of these are really timeline operations that in a traditional setting a "camera man" or film editor would manage to emulate more easily using the film footage rather than creating new or photocopied drawings, I think <Frame> is correctly used instead of <Drawing> except perhaps when you delete all the drawings since Pencil2D is really are removing both the frame and the drawing from the program altogether, but that's just something to have in mind; let's leave it as <Frame> :slightly_smiling_face:

Reply:
I think manipulateFrames is fine. Ideally (this is something that can be left for later) this would be a either turned into multiple actions or a submenu submenu containing copy, move, etc. that would automatically open the dialog on that tab.